moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -384,6 +406,7 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
       editComponent: props => (
         <DateFieldEditComponent
           {...props}
+          tableType="Milestone"
```

Comment:
what does `tableType` do?

Reply:
Hey Chia thanks for catching that! `tableType ` was a prop I was using in one of my attempts in solving the problem. I'm going to create a new commit that removes that.